cx2t/src/claw/tatsu/directive/common/Directive.java

```diff
@@ -74,23 +77,37 @@ public static int generateLoopSeq(XcodeProgram xcodeml,
       XnodeUtil.safeDelete(noDependency);
 
       // Debug logging
-      // TODO generic message for OpenMP as well
+      String prefix = "";
+      switch(currentDirective) {
```

Comment:
Could be a good idea to move this to `CompilerDirective.java`... We might need this somewhere else and we do not want to do the switch/case everywhere. 

Something like:
```java
CompilerDirective(String code, String prefix, String debugPrefix) {
    _code = code;
    _prefix = prefix;
    _debugPrefix = debugPrefix;
}

public String getDebugPrefix() {
    return _debugPrefix;
}
```